docs/workflow/blocks/gosub.rst

```diff
@@ -11,12 +11,6 @@ be referenced from a form's ``uiSchema``. Or, creating dashboards
 that are made up of multiple other workflows combined with a ``multi``
 multiplex workflow block.
 
-.. code-block:: json
 
-    {
-      "type": "gosub",
-      "adapterName": "kendraio",
-      "workflowId": "myWorkflow"
-    }
```

Comment:
Did you mean to delete this?

Reply:
I did... because the block editor no longer lets you enter the data in that form. Like our Mapping block, it provides that lower level of structure. 

I can restore it, but put it in a different section.